src/upconversion.rs

```diff
@@ -7,12 +7,13 @@ use crate::shared_types::IntegerOrFloat;
 use crate::ufo::{Groups, Kerning};
 use crate::Error;
 
-/// Convert kerning groups and pairs from v1 and v2 informal conventions to v3 formal conventions.
-/// Converted groups are added (duplicated) rather than replacing the old ones to preserve all data
-/// that external entities might rely on. Kerning pairs are updated to reflect the new group names.
+/// Convert kerning groups and pairs from v1 and v2 informal conventions to
+/// v3 formal conventions. Converted groups are added (duplicated) rather than
+/// replacing the old ones to preserve all data that external entities might
+/// rely on. Kerning pairs are updated to reflect the new group names.
 ///
-/// This is an adaptation from the fontTools.ufoLib reference implementation. It will not check if
```

Comment:
Is this an accidental reformatting?

Reply:
heh not exactly. I use vim, and it's set to 80 columns, and longer lines wrap in away that makes them hard to read; I had a failing test case here and was distracted trying to read the docs, so I wrapped them to 80 columns. 🤷